evalml/tests/automl_tests/test_automl.py

```diff
@@ -4888,7 +3997,7 @@ def test_automl_drop_unknown_columns(columns, AutoMLTestEnv, X_y_binary, caplog)
         y_train=y,
         problem_type="binary",
         optimize_thresholds=False,
-        max_batches=2,
+        max_batches=3,
```

Comment:
Added this for full coverage in `DefaultAlgorithm` and this could benefit from `MockAlgorithm` as well. Like the comment above, I believe the actual pipelines should be tested in the algorithm tests but this was a nice case where it worked with both algorithms.